python/semantic_kernel/connectors/ai/azure_ai_inference/azure_ai_inference_settings.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import ClassVar
+
+from pydantic import SecretStr
+
+from semantic_kernel.kernel_pydantic import HttpsUrl, KernelBaseSettings
+
+
+class AzureAIInferenceSettings(KernelBaseSettings):
+    """Azure AI Inference settings.
+
+    The settings are first loaded from environment variables with
+    the prefix 'AZURE_AI_INFERENCE_'.
+    If the environment variables are not found, the settings can
+    be loaded from a .env file with the encoding 'utf-8'.
+    If the settings are not found in the .env file, the settings
+    are ignored; however, validation will fail alerting that the
+    settings are missing.
+
+    Optional settings for prefix 'AZURE_AI_INFERENCE_' are:
```

Comment:
The `endpoint` and `api_key` attributes are shown as required, but the doc strings shows as Optional?

Reply:
I noticed there are many places in other files that have the same issue. I will fix all I can find.